lib/beaker/dsl/helpers.rb

```diff
@@ -1340,6 +1340,38 @@ def fact(name, opts = {})
         fact_on(default, name, opts)
       end
 
+      # Snapshot provided host
+      #
+      # Only vagrant hosts are supported
+      # and you must have https://github.com/scalefactory/vagrant-multiprovider-snap installed
+      #
+      # @param host [Host] The host to snapshot
+      # @param name [String] The name to use as the snapshot name
+      def take_snapshot_on(host, name)
+        host.do_take_snapshot(name)
```

Comment:
@anodelman Thanks for your feedback.  I was pretty sure I was going about things the wrong way, but I couldn't figure out how else I could get access to the hypervisor object from a host.

> we'll also need to make the hypervisor hash controlled by beaker available to those methods

How might you suggest this is done?  The hash is an instance variable of the NeworkManager object.  Will I need to patch rspec-beaker and add the hypervisor hash as a 'custom setting' there? (or have I completely misunderstood)?